plugins/hdfsviewer/src/azkaban/viewer/hdfs/HdfsBrowserServlet.java

```diff
@@ -337,7 +337,8 @@ private void displayDirPage(FileSystem fs, String user,
       }
       page.add("dirsize", size);
     } catch (AccessControlException e) {
-      page.add("error_message", "Permission denied: " + e.getMessage());
+      String error_message = props.getString("configurable.error.access.message");
```

Comment:
Define the key name in a central location and as a constant?

Please follow the config key naming conventions. See new keys added recently for examples.